vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc9110/HttpContentEncoding.kt

```diff
@@ -0,0 +1,17 @@
+package at.asitplus.wallet.lib.data.rfc9110
+
+import kotlin.jvm.JvmInline
+
+@JvmInline
+value class HttpContentEncoding(val value: String) {
+    companion object {
+        fun validate(value: String) {
+            // TODO()
```

Comment:
TODO ... and better leave RFC 9110 aside?

Reply:
removed